src/plugins/telemetryTable/components/table.vue

```diff
@@ -462,21 +473,34 @@ export default {
         calculateColumnWidths() {
             let columnWidths = {};
             let totalWidth = 0;
-            let sizingRowEl = this.sizingTable.children[0];
-            let sizingCells = Array.from(sizingRowEl.children);
             let headerKeys = Object.keys(this.headers);
+            let sizingTableChildren = Array.from(this.sizingTable.children);
 
             headerKeys.forEach((headerKey, headerIndex)=>{
-                let cell = sizingCells[headerIndex];
-                let columnWidth = cell.offsetWidth;
-                columnWidths[headerKey] = columnWidth;
-                totalWidth += columnWidth;
+                sizingTableChildren.forEach(sizingRowEl => {
```

Comment:
Recommend not having nested loops.

Reply:
Simplified it, does not need any nested loops anymore